tests/conftest.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
```

Comment:
Rough implementation of the functionality provided by run_tests.py. Not complete. This is a work in progress, but totally usable. All tests pass.

Reply:
I'd like to keep this in my branch while I work on the feature, as I am using it. I've become addicted to running out tests with `--pdb`, which breaks the execution when an uncaught exception escapes from a test. I don't mind putting these two files on ice however once I am done.

To be clear, I'm worried if I make those two files local only and not part of the branch that I'll wipe them by mistake and lose them! :p

Reply:
would be great if we could merge it as a separate thing.